client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -184,6 +184,23 @@ export default {
     isNew () {
       return this.module.moduleID === NoID || this.field.fieldID === NoID
     },
+
+    selectOptions () {
+      const options = [
```

Comment:
Why redefine it if you have it already defined in the data.

Instead of filtering like this, which then needs to be amended if a new option is added, do a property on the option object that will indicate if its only usable if multi. And then you can filter by that always.